openstack/ces/v1/alarms/requests.go

```diff
@@ -0,0 +1,120 @@
+package alarms
+
+import (
+	"fmt"
+
+	golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
+)
+
+type ListAlarmsRequest struct {
```

Comment:
Common name for options container structure is `...Opts`, `ListAlarmOpts` in this case